### PR TITLE
remove demo

### PR DESCRIPTION
**Pull request by severo**
The code now lives at https://github.com/hyparam/hyperparam-cli/tree/master/apps/hyparquet-demo

### CODE REVIEW

**Comment by severo** [comment]
See https://github.com/hyparam/hyperparam-cli/pull/51 as a first step

File: package.json [hunk 1]
```diff
@@ -21,17 +21,10 @@
...
     "test": "vitest run"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "28.0.1",
-    "@rollup/plugin-node-resolve": "15.3.0",
-    "@rollup/plugin-replace": "6.0.1",
-    "@rollup/plugin-terser": "0.4.4",
-    "@rollup/plugin-typescript": "12.1.1",
     "@types/node": "22.8.6",
     "@types/react": "18.3.12",
```
Comment (platypii) [inline package.json:29]: I think we can remove react, react-dom and types too, right?

File: README.md [hunk 2]
```diff
@@ -22,7 +22,9 @@ Online parquet file reader demo available at:
 
 https://hyparam.github.io/hyparquet/
 
-[![hyparquet demo](demo/assets/demo.png)](https://hyparam.github.io/hyparquet/)
+[![hyparquet demo](./demo.png)](https://hyparam.github.io/hyparquet/)
```
Comment (platypii) [inline README.md:25]: Once the cli pages are deployed, lets update this link to the hyparquet demo there.

**Review by platypii — APPROVED** [review]
Approved, pending deployment of the cli pages, and updating the link. 👍

File: package.json (hunk 1 above)
Comment (severo) [reply package.json:29]: sure

Comment (severo) [reply package.json]: fixed

**Pull request merged**